doc/source/user/gui.rst

```diff
@@ -0,0 +1,33 @@
+.. _gui:
+
+===
+GUI
+===
+
+A web based GUI is available for jobflow remote.
+Being based on the `FastHTML framework <https://docs.fastht.ml/>`_,
+requires the installation of the ``python-fasthtml`` package::
+
+    pip install python-fasthtml
+
+.. warning::
```

Comment:
Maybe mention that alternatively, it can be installed using pip install jobflow_remote[gui] ?

Reply:
done